packages/design-system/collapsible-tab-view/@types/react-stickynode/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module "react-stickynode";
```

Comment:
FYI: now If some packages are of the wrong type, run `bit tag ...` will not have a `dist` folder. 🤯  and this problem has blocked me for a long time.
I'm not sure if it's the reason for the recent `bit` version, or maybe use other commond to avoid it.